Paper.tex

```diff
@@ -413,17 +413,45 @@ \subsection{The Transaction} \label{subsec:transaction}
 \mathbb{B}_{0} & \text{otherwise}\end{cases}
 \end{equation}
 
+\subsection{The Withdrawal}\linkdest{withdrawal}\label{subsec:The_Withdrawal}
+A withdrawal (formally, \textit{W}) is a tuple of data describing a consensus layer validator's withdrawal of some amount of its staked Ether.
```

Comment:
FYI, not a big issue, `W` is already defined in the paper as `W(w, μ)`

Reply:
We don't have enough letters in greek & latin alphabet 😅
As you said, I think it's not a big issue, `W` for **W**ithdrawal is easy to remember, it follows the same logic as `T` for **T**ransaction, etc. Also, there is already `H` for block **H**eader even if the normal halting function is defined as `H(μ, I)`